tests/plugins/test_gather_builds_metadata.py

```diff
@@ -72,4 +89,23 @@ def test_gather_builds_metadata(workflow):
             "output": [],
         },
     }
+
+    if has_s390x_build_logs:
+        log_file_metadata = output["s390x"]["output"].pop()
+        assert_log_file_metadata(
+            log_file_metadata,
+            expected_filename=build_log_file.name,
+            expected_platform="s390x",
+        )
+
+        upload_file_info = {
+            "local_filename": str(build_log_file),
+            "dest_filename": build_log_file.name,
+        }
+        assert [upload_file_info] == workflow.data.koji_upload_files
+    else:
+        assert re.search(r"not found: .+s390x-build.log", caplog.text, flags=re.MULTILINE)
```

Comment:
small detail: I think we don't need MULTILINE, it only affects the behavior of `^` and `$`

Reply:
@chmeliik You are right :+1: 

Updated. Since only these two lines change, once the CI passes, I'll merge this PR.